docs/csharp/programming-guide/arrays/codesnippet/CSharp/passing-arrays-as-arguments_4.cs

```diff
@@ -1,68 +1,57 @@
-class ArrayClass
+using System;
+
+class ArrayExample
 {
     static void PrintArray(string[] arr)
     {
         for (int i = 0; i < arr.Length; i++)
         {
-            System.Console.Write(arr[i] + "{0}", i < arr.Length - 1 ? " " : "");
+             Console.Write(arr[i] + "{0}", i < arr.Length - 1 ? " " : "");
         }
```

Comment:
The whole `for` loop can be replaced with the call to [string.Join](https://docs.microsoft.com/en-us/dotnet/api/system.string.join?view=netcore-2.1#System_String_Join_System_String_System_String___). That makes the method one-liner:
```csharp
static void PrintArray(string[] arr) => Console.WriteLine(string.Join(" ", arr));
```